docs/gh-pages-build.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/local/bin/node
+/**
+ * Build a release for the gh-pages docs site.
+ */
+
+var fs = require('fs');
+
+var execSync = require('child_process').execSync;
+
+var usage = '\nbuild <vn.n.n>\n';
+
+// Read the command-line args
+var args = process.argv;
+
+if (args.length < 3) {
+  exit(usage);
+}
+
+version = args[2];
+if (!version.match(/v\d{1,2}.\d{1,2}.\d{1,2}/)) {
+  exit(usage);
+}
+
+// Exit with a message
+function exit(message) {
+  console.log(message,'\n');
+  process.exit();
+};
+
+// Exec with echo
+function execho(command) {
+  console.log(command);
+  try {
+    execSync(command, {stdio: 'inherit'});
+  } catch (error) {
+   process.exit(error.status);
+ }
+};
+
+// Add new version number to versions.js
+var versions = JSON.parse(fs.readFileSync('./src/www/versions.json', 'utf8'));
+versions.unshift(version);
+fs.writeFileSync('./src/www/versions.json', JSON.stringify(versions))
```

Comment:
What about having a pretty stringify?
This way, the git history will be easier to read (one version per line)